master_xml.py

```diff
@@ -295,6 +295,18 @@ def evaluateTokens(cards):
                         cards.get(tokenId)['released'] = True
                         card['related'].append(tokenId)
 
+def evaluateKeywords(cards):
+    for cardId in cards:
+        card = cards[cardId]
+        if card.get('infoRaw') == None:
+            continue
+        card['keywords'] = []
+        # Find all keywords in info string. E.g. find 'spawn' in '<keyword="spawn">'
+        # Can just use en-US here. It doesn't matter, all regions will return the same result.
+        result = re.findall(r'.*?\<keyword=(.*?)\>.*?', card['infoRaw']['en-US'])
```

Comment:
not an issue, but the regex could be cleaner (no `.*?` or unnecessary escape `\<`)
```js
/<keyword=([^>]+)>/
```